### PR TITLE
fix(subgraph): include top 20 VIRTUAL/* pools in V2 Base subgraph

### PR DESCRIPTION
**Pull request by xrsv**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Allow top 20 VIRTUAL/* pool for V2 on Base subgraph pool filtering.

### CODE REVIEW

File: src/providers/v2/subgraph-provider.ts [hunk 1]
```diff
@@ -317,4 +314,58 @@ export class V2SubgraphProvider implements IV2SubgraphProvider {
...
+    return (
+      this.chainId === ChainId.BASE &&
+      V2SubgraphProvider.VirtualPairBaseV2PoolIds.has(pool.id.toLowerCase())
+    );
+  }
+
+  // TODO: Remove. Temporary fix to allow VIRTUAL/* V2 top pools to be included in the list until the subgraph is re-indexed.
+  private static readonly VirtualPairBaseV2PoolIds: Set<string> = new Set([
+    // Pool ID for VIRTUAL / GAME
+    '0xd418dfe7670c21f682e041f34250c114db5d7789',
```
Comment (cgkol) [inline src/providers/v2/subgraph-provider.ts:329]: do we want all these pools, or do we want instead to have just virtual/eth? or will the other one not be found at all anyway so we at least need these?

Comment (xrsv) [reply src/providers/v2/subgraph-provider.ts:330]: Yes we need those. `VIRTUAL/ETH` already goes through filtering.But it seems that usually best path for `ETH/{VirtualEcosystemToken}` swap goes through VIRTUAL, e.g. `ETH/GAME` swap -> `ETH/VIRTUAL/GAME` so we need to allow top `{VirtualEcosystemToken}/VIRTUAL` pools until subgraph re-indexes which might take a while.

**Review by jsy1218 — APPROVED** [review]
We need to ask subgraph team to automate calculating tracked eth TVL

File: src/providers/v2/subgraph-provider.ts (hunk 1 above)
Comment (cgkol) [reply src/providers/v2/subgraph-provider.ts:330]: got it, understood

**Pull request merged**